### DISCUSSION TITLE
Pen Test Planner

### DISCUSSION THREAD

**Discussion by bthird** [pwndoc/pwndoc#221] [Ideas]
Would it be possible to add a simple task driven planner to the report tool. I sometimes have complex pentests which require multiple systems to be tested and owners to be on standby in case there are any ill affects. 

Obviously I can use a separate tool but if the client doesn't provide access/credentials or incurs a delay it would be great to add that to the report. 

The main aim is to cover my back in case of delays during testing and also to inform the client (and myself) when tasks need to be completed. If possible tracking multiple clients as well would be very helpful as it helps me make the decision of whether I take on extra work.

**Comment by yeln4ts** [comment]
The goal is to stay focus on the reporting part of a pentest. 
Of course you could always add a custom section that will include additional information like planning and credentials. This section could then be included or not in the report at your discretion.
